src/foam/nanos/script/Script.js

```diff
@@ -126,7 +137,7 @@ foam.CLASS({
           shell.set("currentScript", this);
           shell.set("x", x);
           shell.eval("runScript(String name) { script = x.get(\\"scriptDAO\\").find(name); if ( script != null ) eval(script.code); }");
-          shell.eval("sudo(String user) { foam.util.Auth.sudo(x, user); }");
+          shell.eval("foam.core.X sudo(String user) { foam.util.Auth.sudo(x, (String) user); }");
```

Comment:
Don't you need a return statement here?

Reply:
No that is implied. It automatically returns the value of the the last function call.